Bump zem version

skarahoda opened the pull request.

Commit:
Bump zem version

dirtybit reviewed (commented):
I think the changes on package.json are missing.

Commit:
Manually update package.json

skarahoda commented:
> I think the changes on package.json are missing.

I updated the zem from cli before and it didn't update the `package.json` file. I updated the file manually.
